django_countries/fields.py

```diff
@@ -214,17 +214,20 @@ def __get__(self, instance=None, owner=None):
         if self.field.name not in instance.__dict__:
             instance.refresh_from_db(fields=[self.field.name])
         value = instance.__dict__[self.field.name]
-        if self.field.multiple:
+        if self.field.multiple and not value is None:
             return [self.country(code) for code in value]
         return self.country(value)
 
     def country(self, code):
-        return Country(
+        if code is None:
+            return None
+        else:
+                    return Country(
             code=code, flag_url=self.field.countries_flag_url,
             custom_countries=self.field.countries)
 
     def __set__(self, instance, value):
-        if self.field.multiple:
+        if self.field.multiple and not value is None:
```

Comment:
I'm not sure this is part of the required fix. Is Admin really trying to set the field to None explicitly?